### PR TITLE
Add pyecharts

### PR DESCRIPTION
**Pull request by sunhailin-Leo**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by maximlt** [comment]
Hi all, I'm interested in having pyecharts available on conda-forge. Is there any way I can help to get this merged?

File: recipes/pyecharts/meta.yaml
Comment (xhochy) [inline recipes/pyecharts/meta.yaml:52]: We need approval from these maintainers to be listed.

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/pyecharts/meta.yaml
Comment (sunhailin-Leo) [reply recipes/pyecharts/meta.yaml:52]: This project, at the moment, is mainly maintained by me (sunhailin-Leo).  
A few other maintainers are no longer working on maintenance, but I've kept them on the list of project maintainers out of respect for their contributions, so I hope you can understand.(chenjiandongx, chfw, kinegratii) https://github.com/pyecharts/pyecharts#-author

Comment (xhochy) [reply recipes/pyecharts/meta.yaml:52]: We will only add them as maintainer here if they consent to it. Please remove them otherwise. Conda-forge maintenance and upstream maintenance is independent.

**Comment by Episkiliski** [comment]
Hi @xhochy , what is the blocker right now to have this merged?

**Comment by xhochy** [comment]
> Hi @xhochy , what is the blocker right now to have this merged? 

See my last review comment. Feel free to submit a PR with the changes made by yourself.

**Comment by Episkiliski** [comment]
> > Hi @xhochy , what is the blocker right now to have this merged?
> 
> See my last review comment. Feel free to submit a PR with the changes made by yourself.

Thanks for the reply, unfortunately I don't have the knowledge to do this. @sunhailin-Leo , is this something you can fix?

**Comment by Episkiliski** [comment]
@xhochy , how can we prove that the list of mantainers agree being on this list?. Please elaborate.

With all due respect, I don't understand why this is still blocking the release of the package in conda-forge given that @sunhailin-Leo is the primary mantainer (you just have to check the package repository).

Thanks for your time.

File: recipes/pyecharts/meta.yaml
Comment (sunhailin-Leo) [reply recipes/pyecharts/meta.yaml:52]: Even if the past maintainers I listed aren't working on maintenance right now, or if they aren't available to make approvals right now, that shouldn't stop me, as the current maintainer, from listing them on the repository's maintainer list.

Comment (xhochy) [reply recipes/pyecharts/meta.yaml:52]: This is not about upstream maintenance but about maintenance of this conda package. These are independent things. We don't want to spam anyone without their approval with PR review requests.

**Pull request closed without merging**
